_ingest-pipelines/processors/ml-inference.md

```diff
@@ -150,9 +194,263 @@ The response confirms that, in addition to the `passage_text` field, the process
 Once you have created an ingest pipeline, you need to create an index for ingestion and ingest documents into the index.
 {: .note}
 
----
+### Example: Local model
+
+The following example configures an `ml_inference` processor with a local model.
+
+**Step 1: Create a pipeline**
+
+The following example creates an ingest pipeline for the `huggingface/sentence-transformers/all-distilroberta-v1` local model. The model is a sentence transformer [pretrained model]({{site.url}}{{site.baseurl}}/ml-commons-plugin/pretrained-models/#sentence-transformers) hosted in your OpenSearch cluster. 
+
+If you invoke the model using the Predict API, the request looks like this:
+
+```json
+POST /_plugins/_ml/_predict/text_embedding/cleMb4kBJ1eYAeTMFFg4
+{
+  "text_docs":[ "today is sunny"],
+  "return_number": true,
+  "target_response": ["sentence_embedding"]
+}
+```
+
+Using this schema, you'll specify the `model_input` as follows:
+
+```json
+ "model_input": "{ \"text_docs\": ${input_map.text_docs}, \"return_number\": ${model_config.return_number}, \"target_response\": ${model_config.target_response} }"
+```
+
+In the `input_map`, you'll map the document `book.*.chunk.text.*.context` field to the `text_docs` field expected by the model:
+
+```json
+"input_map": [
+  {
+    "text_docs": "book.*.chunk.text.*.context"
+  }
+]
+```
+
+Because you're specifying the field to convert into embeddings as a JSON path, you need to set the `full_response_path` to `true` so that the full JSON document is parsed to obtain the input field:
```

Comment:
```suggestion
Because you specified the field to convert into embeddings as a JSON path, you need to set the `full_response_path` to `true` so that the full JSON document is parsed to obtain the input field:
```